test/odbc/psqlodbc/psqlodbc_tests_common.cpp

```diff
@@ -0,0 +1,598 @@
+  }
+
+  for (int i = 0; i < updatedValues.size(); i++) {
+
+    rcode = SQLExecDirect(odbcHandler.GetStatementHandle(), (SQLCHAR *)UpdateTableStatement(tableName, update_col, 
+      UPDATE_WHERE_CLAUSE).c_str(), SQL_NTS);
+    EXPECT_EQ(rcode, SQL_ERROR);
+    odbcHandler.CloseStmt();
+
+    // Assert that no values changed
+    odbcHandler.ExecQuery(SelectStatement(tableName, {"*"}, vector<string>{orderByColumnName}));
+    rcode = SQLFetch(odbcHandler.GetStatementHandle());
+
+    EXPECT_EQ(rcode, SQL_SUCCESS);
+    EXPECT_EQ(pk_len, INT_BYTES_EXPECTED);
+    EXPECT_EQ(pk, pkValue);
+    EXPECT_EQ(data_len, expectedInsertedValues[i].size());
+    EXPECT_EQ(data, expectedInsertedValues[i]);
+
+    rcode = SQLFetch(odbcHandler.GetStatementHandle());
+    EXPECT_EQ(rcode, SQL_NO_DATA);
+
+    odbcHandler.CloseStmt();
+  }
+}
+
+void testPrimaryKeys(ServerType serverType, const string &schemaName, const string &pkTableName, const vector<string> &primaryKeyColumns) {
+  OdbcHandler odbcHandler(Drivers::GetDriver(serverType));
+  odbcHandler.Connect(true);
+
+  RETCODE rcode;
+
+  char table_name[BUFFER_SIZE];
+  char column_name[BUFFER_SIZE];
+  int key_sq{};
+  char pk_name[BUFFER_SIZE];
+
+  const vector<tuple<int, int, SQLPOINTER, int>> constraints_bind_columns = {
+    {3, SQL_C_CHAR, table_name, BUFFER_SIZE},
+    {4, SQL_C_CHAR, column_name, BUFFER_SIZE},
+    {5, SQL_C_ULONG, &key_sq, BUFFER_SIZE},
+    {6, SQL_C_CHAR, pk_name, BUFFER_SIZE}
+  };
+  ASSERT_NO_FATAL_FAILURE(odbcHandler.BindColumns(constraints_bind_columns));
+
+  rcode = SQLPrimaryKeys(odbcHandler.GetStatementHandle(), NULL, 0, (SQLCHAR *)schemaName.c_str(), SQL_NTS, (SQLCHAR *)pkTableName.c_str(), SQL_NTS);
+  EXPECT_EQ(rcode, SQL_SUCCESS);
+
+  int curr_sq{0};
+  for (auto columnName : primaryKeyColumns) {
+    ++curr_sq;
+    rcode = SQLFetch(odbcHandler.GetStatementHandle());
+    EXPECT_EQ(rcode, SQL_SUCCESS);
+
+    EXPECT_EQ(string(table_name), pkTableName);
+    EXPECT_EQ(string(column_name), columnName);
+    EXPECT_EQ(key_sq, curr_sq);
+  }
+  rcode = SQLFetch(odbcHandler.GetStatementHandle());
+  EXPECT_EQ(rcode, SQL_NO_DATA);
+  odbcHandler.CloseStmt();
+}
+
+void testUniqueConstraint(ServerType serverType, const string &tableName, const vector<string> &uniqueConstraintColumns) {
+  OdbcHandler odbcHandler(Drivers::GetDriver(serverType));
+  odbcHandler.Connect(true);
+
+  char columnName[CHARSIZE];
+  RETCODE rcode;
+
+  const vector<tuple<int, int, SQLPOINTER, int>> constraints_bind_columns = {
+    {1, SQL_C_CHAR, columnName, CHARSIZE}
+  };
+  ASSERT_NO_FATAL_FAILURE(odbcHandler.BindColumns(constraints_bind_columns));
+
+  const string UNIQUE_QUERY = 
+    "SELECT C.COLUMN_NAME FROM "
+    "INFORMATION_SCHEMA.TABLE_CONSTRAINTS T "
+    "JOIN INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE C "
+    "ON C.CONSTRAINT_NAME=T.CONSTRAINT_NAME "
+    "WHERE "
+    "C.TABLE_NAME='" + tableName.substr(tableName.find('.') + 1, tableName.length()) + "' "
```

Comment:
Rather than `tableName.substr(tableName.find('.') + 1, tableName.length())`
We can just use the passed in `tableName`, so in other words, we just pass in the object name without the database or schema

Changing it to use the parameter will also help later with the MSSQL (not for unique key specifically, but example for Primary Key)
PG wants -> `master_dbo`
BBF wants -> `dbo`

Reply:
Done. I will make a comment in the function signature indicating that we should pass the object name without the database or schema.

For primary keys, I already have it so that you need to pass in schema & the table name (without DB or schema) to the function call. I will document this.